NewForce-at-Mountwest/bangazon-api-gnocci
# commit to add models for tickets

# Description

Added Following Classes to Model:
Department
Employee
Computer
ComputerEmployee

## Related Ticket(s)
Ticket #6

## Steps to Test
Please reference Ticket #6 (https://trello.com/c/Zw77d33Y/6-ticket-6-allow-developers-to-access-the-employee-resource) to to confirm correct usage of class items. 
Run Code 
`````
Ctrl + F5
`````
to check for build errors. 

# Checklist:
